libs/utils/utils.js

```diff
@@ -1391,38 +1391,82 @@ export function enablePersonalizationV2() {
   const enablePersV2 = getMepEnablement('personalization-v2');
   return !!enablePersV2 && isSignedOut();
 }
+export function getCookie(key) {
+  const cookie = document.cookie.split(';')
+    .map((x) => decodeURIComponent(x.trim()).split(/=(.*)/s))
+    .find(([k]) => k === key);
+  return cookie ? cookie[1] : null;
+}
+export function normCountry(country) {
+  if (!country) return null;
+  let c = country?.toLowerCase();
+  if (c === 'uk') c = 'gb';
+  return c.split('_')[0];
+}
+export async function getC0002(optin, akamaiLocale) {
```

Comment:
We are running these methods for everyone, regardless of the user location, privacy initialization or consent categories. My suggestion is to write the code in a way that code is executed (and even imported) only if needed.